package.json

```diff
@@ -4,7 +4,11 @@
   "description": "A timer for mob programming and pair programming.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "cypress:ci": "cypress run",
+    "cypress:dev": "cypress open",
+    "start-server": "grunt",
+    "test:dev": "start-server-and-test start-server http://localhost:8080 cypress:dev",
+    "test:ci": "start-server-and-test start-server http://localhost:8080 cypress:ci"
```

Comment:
I tried to do this through Grunt, since it's what the project uses for all it's tasks, but I couldn't do it. 

For Cypress to work, you need to run both the dev server and Cypress server in parallel. However, for parallel tasks support in Grunt, it seems you need to install a plugin, so I used this `start-server-and-test` library instead, which seemed to be simpler. Let me know what you think about it. 

Once we get this task-running-setup-thing straight, if it interests you, we can also create a GitHub Action to run the tests (and other tasks, such as linting) against each pull request.